templates/vue/src/components/tyde/TydeLog.vue

```diff
@@ -54,16 +72,29 @@ export default {
   padding: 24px;
   width: 100%;
   display: flex;
-  
-  img {
-    max-width: 250px;
-  }
 
-  .thumbnail-placeholder {
+  .log-thumbnail {
     width: 250px;
-    height: 250px;
-    background-color: gray;
+    height: 225px;
     float: left;
```

Comment:
Sorry I might have missed this is an earlier PR, but I think we should try to stick to using `flexbox` or css grid over `float`. It's clearer what the styling does that way.

Reply:
I didn't touch the backpack layout in this PR (I might have done it in other PR, but really don't think so). Though I am not sure if this is an issue provided that it works. Or we can make a separate task to look into this.

Reply:
I think I may have made this change as I was testing it and it was not working for me. I agree with Andrew that we should make a separate task to fix this - maybe along with the SCSS variable changes.

Reply:
That sounds good to me.